examples/todos/src/containers/VisibleTodoList.js

```diff
@@ -1,3 +1,5 @@
+// @flow
```

Comment:
Missing type here. Basically, we'll try to apply type for all methods/functions in module 😄